www/src/components/layer-model/layer-model.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react"
+import hex2rgba from "hex2rgba"
+
+import { space, colors, radii } from "../../utils/presets"
+import LayerIcon from "../../assets/icons/layer-icon"
+import {
+  ContentLayerContent,
+  BuildLayerContent,
+  DataLayerContent,
+  ViewLayerContent,
+  AppLayerContent,
+} from "./layer-content-sections"
+
+const Layer = ({ layer, onClick, selected, children }) => {
+  const { baseColor, title } = layer
+
+  return (
+    <button
+      onClick={onClick}
+      css={{
+        cursor: `pointer`,
+        borderRadius: radii[3],
+        padding: space[2],
+        color: colors.grey[60],
+        fontWeight: selected ? `bold` : `normal`,
+        backgroundColor: colors.grey[5],
+        border: selected
+          ? `2px ${colors[baseColor][60]} solid`
+          : `2px transparent solid`,
+        ":focus": {
+          outline: 0,
+          boxShadow: `0 0 0 3px ${hex2rgba(colors[baseColor][30], 0.5)}`,
+        },
+        ":hover": {
+          backgroundColor: colors[baseColor][5],
+        },
+      }}
+    >
+      <div
```

Comment:
Can these items inside the button be changed to `<span>` elements and styled with CSS to avoid putting DIVs inside?